api_service.py

```
@@ -306,6 +369,35 @@ def get_account_service() -> AccountCategoryClientInterface:
+            )
+
+            # Initialize category aggregation config
+            config = CategoryAggregationConfig()
+
+            # Create recommendation service
+            recommendation_service = BlockingRecommendationService(
+                repository=settings_service.repository,
+                config=config,
+                domain_service=domain_service
+            )
+        except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)